codec/src/types/primitives.rs

```diff
@@ -40,6 +43,41 @@ impl_numeric!(i128, get_i128, put_i128);
 impl_numeric!(f32, get_f32, put_f32);
 impl_numeric!(f64, get_f64, put_f64);
 
+// Usize implementation
+// Since most values refer to a length or size of an object in memory, values are usually biased
```

Comment:
I recommend putting this somewhere that gets rendered. This seems way too important for an in-code only comment.